x-pack/plugins/security_solution/common/endpoint/models/event.ts

```diff
@@ -52,3 +52,136 @@ export function parentEntityId(event: ResolverEvent): string | undefined {
   }
   return event.process.parent?.entity_id;
 }
+
+/**
+ * @param event The event to get the category for
+ */
+export function primaryEventCategory(event: ResolverEvent): string {
+  // Returning "Process" as a catch-all here because it seems pretty general
+  let eventCategoryToReturn: string = 'Process';
```

Comment:
please remove the `'Process'` or `''` defaults here. My expectation reading this is that the method would show some data from the event.

Reply:
https://github.com/elastic/kibana/pull/69098/commits/436ae12c77be6ae2f3a2981daaf8d3b4d773f23c